src/core/config/mesh_forwarder.h

```diff
@@ -148,4 +150,56 @@
 #endif
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_TX_QUEUE_STATISTICS_ENABLE
+ *
+ * Define as 1 to enable TX queue time-in-queue statistics collection feature.
+ *
+ * When enabled, histogram of the time-in-queue of messages in the transmit queue is collected. The time-in-queue is
+ * tracked for direct transmissions only and is measured as the duration from when a message is added to the transmit
+ * queue until it is passed to the MAC layer for transmission or dropped.
+ *
+ * The histogram data consists of number of bins, each representing a range of time-in-queue values. The bin interval
+ * length is specified by the `OPENTHREAD_CONFIG_TX_QUEUE_STATISTICS_HISTOGRAM_BIN_INTERVAL` configuration, and the
+ * maximum tracked interval is given by the `OPENTHREAD_CONFIG_TX_QUEUE_STATISTICS_HISTOGRAM_MAX_INTERVAL`.
+ *
+ * Along with histogram, the maximum observed time-in-queue is also tracked.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_TX_QUEUE_STATISTICS_ENABLE
+#define OPENTHREAD_CONFIG_TX_QUEUE_STATISTICS_ENABLE (OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE && OPENTHREAD_FTD)
```

Comment:
Do we want `BORDER_ROUTER` or `BORDER_ROUTING` here? :D

Reply:
Changed to `BORDER_ROUTER`.